streams/src/main/java/org/apache/kafka/streams/state/internals/CachingKeyValueStore.java

```diff
@@ -95,18 +94,6 @@ QueryResult<?> apply(
         this.timestampedSchema = timestampedSchema;
     }
 
-    @SuppressWarnings("deprecation") // This can be removed when it's removed from the interface.
-    @Deprecated
-    @Override
-    public void init(final ProcessorContext context,
-                     final StateStore root) {
-        initInternal(asInternalProcessorContext(context));
```

Comment:
If we call `initInternal` only in a single place now, we might consider inlining it?

Reply:
agree. @xijiu please apply "inline" to elsewhere, too